PluginsAndFeatures/azure-toolkit-for-intellij/resources/azurefunction/templates/project/pom_xml.template

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>$(groupId)</groupId>
+    <artifactId>$(artifactId)</artifactId>
+    <version>$(version)</version>
+    <packaging>jar</packaging>
+
+    <name>Azure Java Functions</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <azure.functions.maven.plugin.version>1.4.0</azure.functions.maven.plugin.version>
```

Comment:
latest is 1.4.1, besides, do we need a pipeline to update the dependency versions automatically?

Reply:
will be handled later